src/vs/workbench/browser/actions/developerActions.ts

```diff
@@ -139,17 +139,28 @@ class ToggleScreencastModeAction extends Action {
 		}));
 
 		const keyboardMarker = append(container, $('.screencast-keyboard'));
+
 		disposables.add(toDisposable(() => keyboardMarker.remove()));
 
+		const updateKeyboardFontSize = () => {
+			keyboardMarker.style.fontSize = `${this.configurationService.getValue<number>('screencastMode.fontSize') || 56}px`;
```

Comment:
At the moment this event is being fired even if the entered value fails validation (ex: `300`) so I think that it would be good to check the value of the setting within the event handler and only apply it if it is between the min and max values.